lib/text.rb

```diff
@@ -0,0 +1,20 @@
+require 'twilio-ruby'
+require_relative '../.env.rb'
+
+class Text
+
+  attr_reader :customer_number
+
+  def self.make_text (customer_number, message )
```

Comment:
Do not put a space between a method name and the opening parenthesis.<br>Space inside parentheses detected.